templates/content/_fields.html.twig

```diff
@@ -1,12 +1,13 @@
 {% for group in groups if group != 'Relations' %}
+
     <div class="tab-pane {% if loop.first %}show active{% endif %}" id="{{ group|slug }}" role="tabpanel"
          aria-labelledby="{{ group|slug }}-tab">
         {% for key, fielddefinition in record.definition.fields if fielddefinition.group == group %}
 
             {% if record.hasField(key) %}
                 {% set field = record.getField(key)|translated(currentlocale) %}
             {% else %}
-                {% set field = field_factory(key, fielddefinition) %}
+                {% set field = record|field_factory(key, fielddefinition) %}
```

Comment:
it should be a function, not filter

Reply:
Reverted this bit. It's it's own function again.